### ISSUE TITLE
Insight Tag

### ISSUE THREAD

**Issue by wcsu95** [credmark/credmark-website#134]
@elektramurphy 
I forwarded the insight tag installation instruction via e-mail. Can you help get it up? It's to track where people are coming in from for our ad.

**Comment by elektramurphy** [comment]
@wcsu95 sorry I missed this, do you still want this setup?

**Comment by wcsu95** [comment]
@elektramurphy 
Yes, we'll need this for LinkedIn and twitter (in case) we are able to use it.

**Comment by elektramurphy** [comment]
@wcsu95 is this ticket still relevant?

**Comment by wcsu95** [comment]
> @wcsu95 is this ticket still relevant?

Yes, let's add the tags so we can track if ads are ran in the future.

**Comment by wcsu95** [comment]
> @wcsu95 is this ticket still relevant?

@elektramurphy 
Yes, let's add the tags so we can track if ads are ran in the future.
